package.json

```diff
@@ -99,7 +99,7 @@
     "cozy-flags": "2.4.1",
     "cozy-interapp": "0.4.9",
     "cozy-realtime": "3.2.1",
-    "cozy-ui": "40.1.0",
+    "cozy-ui": "^51.8.0",
```

Comment:
Aren't we using fixed versions for cozy libs?

Reply:
Let's discuss that on Tuesday but no ;)